zest/releaser/baserelease.py

```diff
@@ -155,12 +163,32 @@ def _change_header(self, add=False):
         Change the version number and the release date or development status.
 
         @add:
-        - False: edit current header (prerelease)
+        - False: edit current header (prerelease or bumpversion)
         - True: add header (postrelease)
+
+        But in some cases it may not be wanted to change a header,
+        especially when towncrier is used to handle the history.
+        Otherwise we could be changing a date of an already existing release.
+        What we want to avoid:
+        - change a.b.c (1999-12-32) to x.y.z (unreleased) [bumpversion]
+        - change a.b.c (1999-12-32) to x.y.z (today) [prerelease]
+        But this is fine:
+        - change a.b.c (unreleased) to x.y.z (unreleased) [bumpversion]
+        - change a.b.c (unreleased) to a.b.c (today) [prerelease]
+        - change a.b.c (unreleased) to x.y.z (today) [prerelease]
         """
         if self.data['history_file'] is None:
             return
         good_heading = self.data['history_header'] % self.data
+        if not add and self.data.get('has_released_header'):
+            # So we are editing a line, but it already has a release date.
+            logger.warning(
+                'Refused to edit history header, because it looks to be from '
```

Comment:
"... edit THE FIRST history header ...": would that be a bit clearer?

Aaargh, we're using "heading" and "header" interchangeably, it seems. I think, headING is more correct.
Perhaps use that in this warning?

Reply:
Both done.

We seem to use 'header' and 'heading' roughly equal times...

Reply:
That's what I feared :-)